dhis-2/dhis-support/dhis-support-system/src/main/java/org/hisp/dhis/system/grid/ListGrid.java

```diff
@@ -62,7 +63,7 @@
  * @author Lars Helge Overland
  */
 public class ListGrid
-    implements Grid
+    implements Grid, Serializable

```

Comment:
Is this required since Grid already extends Serializable?

Reply:
Not really. I just did it t be on the safe side. Could be removed I guess.